### ISSUE TITLE
Replace exec statement with exec() function in twisted/conch

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8418]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8418|
|Type|enhancement|
|Created|2016-06-04 04:22:55Z|
|Branch|https://github.com/twisted/twisted/tree/8418-rodrigc-exec|

See: https://docs.python.org/3.0/whatsnew/3.0.html#removed-syntax

**Removed keyword: exec() is no longer a keyword; it remains as a function. (Fortunately the function syntax was also accepted in 2.x.) **

In Python 2, you can do this:

```
    exec "foobar"    # WORKS, exec can be invoked as a statement
    exec("foobar")   # WORKS, exec can be invoked as a function
```

In Python 3, you can only do:

```
    exec "foobar"     # SyntaxError !
    exec("foobar")    # WORKS
```

The use of exec as a statement need to be changed to function form
to work with Python 3

<details><summary>Searchable metadata</summary>

```
trac-id__8418 8418
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__8418_rodrigc_exec 8418-rodrigc-exec
branch_author__ 
status__closed closed
resolution__fixed fixed
component__conch conch
keywords__None None
time__1465014175007420 1465014175007420
changetime__1465119632098125 1465119632098125
version__None None
owner__rodrigc rodrigc
cc__z3p
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_2"></a>|@rodrigc commented|
|-|-|

See:
https://github.com/twisted/twisted/pull/138

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_3"></a>|@adiroiban set owner to @rodrigc|
|-|-|

changes look good. thanks.

will wait for buildbot and merge if everything is ok

thanks!

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_4"></a>|@adiroiban commented|
|-|-|

buildbot is back. 100% coverage :) https://codecov.io/gh/twisted/twisted/commit/9a429a6cea898d14c6277d8a2d23e90e70c7a5f0

will merge.

thanks!

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_5"></a>|@adiroiban set status to `closed`|
|-|-|

In changeset b4569d5e28beab05b38ef7a148acca8052a21519
```
#!CommitTicketReference repository="" revision="b4569d5e28beab05b38ef7a148acca8052a21519"
Merge 8418-rodrigc-exec: Use exec function in twisted.conch.

Author: rodrigc
Reviewer: adiroiban
Fixes: #8418
```
